pubspec.yaml

```diff
@@ -65,6 +65,7 @@ dependencies:
   url_launcher: ^6.1.11
   url_launcher_android: ">=6.1.0"
   sqlite3: ^2.4.0
+  video_player: ^2.8.3
```

Comment:
The dependency should be added in its own commit. See `git log --stat -p --full-diff pubspec.*` for examples.